sdk/cdsclient/interface.go

```diff
@@ -227,7 +227,7 @@ type WorkerClient interface {
 	WorkerModelBook(id int64) error
 	WorkerList() ([]sdk.Worker, error)
 	WorkerDisable(id string) error
-	WorkerModelAdd(name string, modelType string, image string, groupID int64) (sdk.Model, error)
+	WorkerModelAdd(name string, modelType string, dockerModel sdk.ModelDocker, vmModel sdk.ModelVirtualMachine, groupID int64) (sdk.Model, error)
```

Comment:
if WorkerModelAdd must use with either a sdk.ModelDocker os sdk.ModelVirtualMachine, the signature must different